leakcanary-android-core/src/main/java/leakcanary/internal/activity/screen/LeakScreen.kt

```diff
@@ -127,7 +128,19 @@ internal class LeakScreen(
     spinner.setSelection(selectedLeakTraceIndex)
   }
 
-
+private fun parseLinks(str: String):String {
+    val words = str.split(" ")
+    var parsedString = ""
+    for (word in words) {
+      parsedString += if (Patterns.WEB_URL.matcher(word).matches()) {
+        "<a href=\"${word}\">${word}</a>";
+      } else {
+        word
+      }
+      if (words.indexOf(word) != words.size - 1) parsedString += " "
+    }
+    return parsedString
+}
```

Comment:
I tried using Linkify. All links work except "Learn more" link.

Reply:
Huh, that's strange. Is it something different about that link? Or it somehow clashes with our own html code? Maybe adding some `<br>` around can fix it?

Reply:
The goal of this PR is to change only the content of `selectedLeak.description` because that's the only place where we can have raw URLs. Linkify requires a Spannable, so I suspect @0x109 took `title` (output of Html.fromHtml) and ran that through Linkify, which means existing URLs that are already in `<a>` tags are getting linkified a second time, hence why Learn More stops working.

Since we only want to linkify `selectedLeak.description`, the only proper way to use Linkify would be to create a spannable for the description, linkify it, then insert that into the final spannable. I can't imagine that being much simpler than the current approach. Also, let's not add random `<br>` tag @Armaxis  ;)